exercises/concept/string-vs-str/.docs/instructions.md

```diff
@@ -1,23 +1,36 @@
-We need a builder for a CSV record per the standard defined in [RFC 4180](https://tools.ietf.org/html/rfc4180).
+Your company has been hired to generate CSV documents from user input.
```

Comment:
This looks like an unintentional semantic change: the builder produces individual CSV records, not full documents.

```suggestion
Your company has been hired to generate CSV records from user input.
```